cmd/mungedocs/mungedocs.go

```diff
@@ -28,6 +29,8 @@ import (
 	flag "github.com/spf13/pflag"
 )
 
+const latestReleaseBranch = "release-1.1"
```

Comment:
> We should change it in the master branch only :)

Is that because this is only ever run from master?  I'd like to suggest that you consider running from the release branch instead.  It's much better to have releases self-contained and self-producing; we've made the mistake of running releases from HEAD of master, and it's kind of ugly.